src/modules/db_operations/seed_trait_files.py

```diff
@@ -61,9 +61,12 @@ def populate_andersenlab_trait_files():
         'species':           get_field_from_record(record, 'Species', nullable=False),
 
         # About trait
-        'description_short': get_field_from_record(record, 'Short_Description'),
-        'description_long':  get_field_from_record(record, 'Long_Description'),
-        'units':             get_field_from_record(record, 'Units'),
+        'trait_name_display_1': get_field_from_record(record, 'Trait_Name_Display1'),
```

Comment:
can we add an `assert()` that all the expected Keys (headers) exist in the sheet as soon as possible and bail out accordingly (show error message about missing keys, case-sensitive keys, etc).

Reply:
Added in [011de8d](https://github.com/AndersenLab/CAENDR/pull/472/commits/011de8d8302fa12cb444e9bbbc04391e38e82746) -- check for set of expected keys, and exit the function with an error if any are missing.  This will terminate the db operation with an error.